lib/election.js

```diff
@@ -818,25 +851,8 @@ function _getManifestFromPeer(
     },
     manifest: callback => api._client.getManifest(
       manifestHash, peerId, callback),
-    checkMajority: ['manifest', (results, callback) => {
-      const manifest = results.manifest;
-      const blockHeight = manifest.blockHeight;
-      const manifestItemCount = manifest.item.length;
-
-      // TODO: consider passing electors through to optimize away this look up
-      api.getBlockElectors(ledgerNode, blockHeight, (err, result) => {
-        if(err) {
-          return callback(err);
-        }
-        const twoThirds = _twoThirdsMajority(result.length);
-        if(manifestItemCount < twoThirds) {
-          return callback(new BedrockError(
-            'The manifest does not contain the required two-thirds majority.',
-            'ValidationError', {manifest, manifestItemCount, twoThirds}));
-        }
-        callback();
-      });
-    }],
+    checkMajority: ['manifest', (results, callback) =>
+      api.checkMajority(results.manifest, ledgerNode, callback)],
```

Comment:
This should only be called for event vote manifests, not event manifests. Event manifests must have at least 1 event hash -- that's it.

Reply:
@dlongley does it seem odd to you that I have never observed this code being called in relation to an event manifest?  It's always RollCall, which is why the tests are passing.

Reply:
I think the test suite isn't constructed so that the nodes ever get different events, right? I think this is something we talked about before and how we need better tests to handle this case.

Reply:
@dlongley actually, I added a 5th block.  In addition to submitting a unique event at each node, I offset the worker start time which I have found creates a great deal of turbulence (by design).